it/src/main/resources/template.conf

```diff
@@ -33,6 +33,7 @@ waves {
         pre-activated-features = {
           2 = 0
         }
+        double-features-periods-after-height = 100000000
```

Comment:
We might tune that later again, I suggest this as "set-to-10k-blocks-at", easier to predict what happens